cli/lib/kontena/cli/stacks/labels/add_command.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Labels
+  class AddCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    parameter "NAME", "Stack name"
+    parameter "LABEL ...", "Labels"
+
+    requires_current_master
+    requires_current_master_token
+
+    def execute
+      original_labels = fetch_master_data(name)
+      update_stack(name, { 'labels' => (original_labels + label_list).uniq })
```

Comment:
Can be `'labels' => original_labels | label_list`

Reply:
✅ RESOLVED